controls/views.py

```diff
@@ -1328,13 +1328,16 @@ def component_library_component(request, element_id):
         # Get the impl_smts contributed by this component to system
         impl_smts = element.statements_produced.filter(statement_type=StatementTypeEnum.CONTROL_IMPLEMENTATION_PROTOTYPE.name)
 
+    inheritances = Inheritance.objects.all()
```

Comment:
Should this always be all??  Or do we need a filter?

Reply:
Yes. There are just three to five records in Inheritance, so really just using DB to allow content to change from install to install.